isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -936,6 +931,88 @@ QString BundleObservation::formatBundleOutputString(bool errorPropagation, bool
 }
 
 
+/**
+ * Returns the list of observation parameter names.
+ *
+ * This will always return at least one set of positions and pointings
+ * because we always output at least the center values even when not solving
+ * for them.
+ *
+ * @return @b QStringList List of observation parameter names
+ */
+QStringList BundleObservation::parameterList() {
+  QStringList paramList;
+
+  // We still want to output the center postion even if not solving for it
+  // so always do at least 1
+  int numberCamPosCoefSolved = std::max(
+      solveSettings()->numberCameraPositionCoefficientsSolved(),
+      1);
```

Comment:
I do not like this but it's needed to preserve functionality. Ideally, this wouldn't be here and the twist would not be there if we weren't solving for it, but that's not the current functionality for the output files.